### DISCUSSION TITLE
Trascrizione del decreto di concessione della cittadinanza: indicazioni casi d'uso di riferimento

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#1042] [Supporto servizi cooperativi] [answered]
La presente per richiedere conferma sul fatto che, al momento, per casi di trascrizione del decreto di concessione della cittadinanza ai sensi:
- dell'articolo 5 della legge 5 febbraio 1992, n. 91
- dell'articolo 9 della legge 5 febbraio 1992, n. 91

debba ancora farsi riferimento al caso d'uso "Trascr_Citt_999".
Se si, è possibile avere notizia sulla previsione di rilascio di casi d'uso specifici a riguardo?
Grazie

**Comment by asqueglia** [comment, marked as answer]
Buon pomeriggio, per pianificare nuovi casi d'uso da prevedere dobbiamo necessariamente fare un passaggio con ANUSCA. Quindi al momento confermiamo la necessità di utilizzare il caso d'uso di servizio lì dove il sistema è manchevole del caso specifico.Gli interventi programmati nel breve periodo sono e saranno specificati sulla roadmap.
